M2/Macaulay2/packages/GameTheory.m2

```diff
@@ -0,0 +1,2711 @@
+newPackage(
+   "GameTheory",
+   Version => "1.0",
+   Date => "May, 2025",
+   Authors => {
+      {Name => "Erin Connelly",
+         Email => "erin.connelly@uni-osnabrueck.de",
+         HomePage => "https://erinconnelly96.github.io/"},
+      {Name => "Vincenzo Galgano",
+         Email => "galgano@mpi-cbg.de",
+         HomePage => "https://sites.google.com/view/vincenzogalgano"},
+      {Name => "Zhuang He",
+         Email => "zhuang.he@unito.it",
+         HomePage => "https://sites.google.com/view/hezhuang/home"},
+      {Name => "Lars Kastner",
+         Email => "kastner@math.tu-berlin.de",
+         HomePage => "https://lkastner.github.io"},
+      {Name => "Giacomo Maletto",
+         Email => "gmaletto@kth.se",
+         HomePage => "https://www.kth.se/profile/gmaletto"},
+      {Name => "Elke Neuhaus",
+         Email => "elke@neuhaus-rp.de",
+         HomePage => "https://sites.google.com/view/elkeneuhaus"},
+      {Name => "Irem Portakal",
+         Email => "mail@irem-portakal.de",
+         HomePage => "https://www.irem-portakal.de"},
+      {Name => "Hannah Tillmann-Morris",
+         Email => "tillmann@mis.mpg.de",
+         HomePage => "https://sites.google.com/view/hannah-tillmann-morris"},
+      {Name => "Chenyang Zhao",
+         Email => "cz2922@ic.ac.uk",
+         HomePage => "https://www.felixzhao.com/"}
+   },
+   Headline => "A package for computing equilibria in game theory",
+   Keywords => {"Game Theory","Equilibria","Nash","Correlated","Dependency","Spohn","Conditional Independence"},
```

Comment:
The Keyword here is meant to put the package into one or more categories from [this page](https://macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_packages_spprovided_spwith_sp__Macaulay2.html). Is there one that you think fits best with this package? If not, we could start a new "Game Theory" category:
```suggestion
   Keywords => {"Game Theory"},
```